kafka/admin/client.py

```diff
@@ -341,6 +343,30 @@ def _find_coordinator_id(self, group_id):
         response = future.value
         return self._find_coordinator_id_process_response(response)
 
+    def _find_many_coordinator_ids(self, group_ids):
+        """Find the broker node_id of the coordinator for each of the given groups.
+
+        Sends a FindCoordinatorRequest message to the cluster for each group_id.
+        Will block until the FindCoordinatorResponse is received for all groups.
+        Any errors are immediately raised.
+
+        :param group_ids: A list of consumer group IDs. This is typically the group
+            name as a string.
+        :return: A list of tuples (group_id, node_id) where node_id is the id
+            of the broker that is the coordinator for the corresponding group.
+        """
+        # Note: Java may change how this is implemented in KAFKA-6791.
```

Comment:
This comment may be outdated, see https://github.com/apache/kafka/pull/4902#issuecomment-390830255

Reply:
Removed both occurrences of that comment.